sympy/solvers/diophantine/diophantine.py

```diff
@@ -3662,67 +3670,78 @@ def sum_of_three_squares(n):
     See Also
     ========
 
-    sum_of_squares()
-    """
-    special = {1:(1, 0, 0), 2:(1, 1, 0), 3:(1, 1, 1), 10: (1, 3, 0), 34: (3, 3, 4), 58:(3, 7, 0),
-        85:(6, 7, 0), 130:(3, 11, 0), 214:(3, 6, 13), 226:(8, 9, 9), 370:(8, 9, 15),
-        526:(6, 7, 21), 706:(15, 15, 16), 730:(1, 27, 0), 1414:(6, 17, 33), 1906:(13, 21, 36),
-        2986: (21, 32, 39), 9634: (56, 57, 57)}
-
-    v = 0
+    power_representation :
+        ``sum_of_three_squares(n)`` is one of the solutions output by ``power_representation(n, 2, 3, zeros=True)``
 
+    """
+    # tuples are sorted; XXX perhaps all 57 of the values of remove(n,4)[0] for n<10**4 that have a single
```

Comment:
for n < 10000, the following dict give {m: s} where m = remove(n, 4)[0] and s is: 
* a single solution {0: (0, 0, 0), 1: (0, 0, 1), 2: (0, 1, 1), 3: (1, 1, 1), 5: (0, 1, 2), 6: (1, 1, 2), 10: (0, 1, 3), 11: (1, 1, 3), 13: (0, 2, 3), 14: (1, 2, 3), 19: (1, 3, 3), 21: (1, 2, 4), 22: (2, 3, 3), 30: (1, 2, 5), 35: (1, 3, 5), 37: (0, 1, 6), 42: (1, 4, 5), 43: (3, 3, 5), 46: (1, 3, 6), 58: (0, 3, 7), 67: (3, 3, 7), 70: (3, 5, 6), 78: (2, 5, 7), 91: (1, 3, 9), 93: (2, 5, 8), 115: (3, 5, 9), 133: (4, 6, 9), 142: (5, 6, 9), 163: (1, 9, 9), 190: (3, 9, 10), 235: (1, 3, 15), 253: (3, 10, 12), 403: (3, 13, 15), 427: (9, 11, 15)}
* the only non-trivial solution {9: (1, 2, 2), 17: (2, 2, 3), 18: (1, 1, 4), 26: (1, 3, 4), 29: (2, 3, 4), 34: (3, 3, 4), 45: (2, 4, 5), 49: (2, 3, 6), 50: (3, 4, 5), 53: (1, 4, 6), 61: (3, 4, 6), 65: (2, 5, 6), 73: (1, 6, 6), 82: (3, 3, 8), 97: (5, 6, 6), 106: (3, 4, 9), 109: (3, 6, 8), 145: (3, 6, 10), 157: (2, 3, 12), 169: (3, 4, 12), 193: (6, 6, 11), 202: (3, 7, 12), 205: (5, 6, 12), 265: (2, 6, 15), 277: (4, 6, 15), 298: (3, 8, 15), 397: (3, 8, 18), 442: (3, 12, 17), 445: (3, 6, 20), 505: (9, 10, 18), 793: (6, 9, 26)}